src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -24,6 +24,33 @@ static uct_md_ops_t uct_rdmacm_md_ops = {
     .is_sockaddr_accessible = uct_rdmacm_is_sockaddr_accessible,
 };
 
+static void uct_rdmacm_print_ip(struct sockaddr *sock_addr)
+{
+    struct sockaddr_in6 *addr_in6;
+    struct sockaddr_in *addr_in;
+    char *ip = NULL;
+
+    switch(sock_addr->sa_family) {
+        case AF_INET:
+            addr_in = (struct sockaddr_in *) sock_addr;
+            ip = ucs_malloc(INET_ADDRSTRLEN, "ipv4 address");
```

Comment:
can use local array of size INET_ADDRSTRLEN

Reply:
but it can also be INET6_ADDRSTRLEN and i wouldnt want to make the declaration inside the case.

Reply:
maybe ucs_max(INET_ADDRSTRLEN, INET6_ADDRSTRLEN)

Reply:
but these are constant values, we know which one is the max, we just dont know which one will be used (it depends on the function's arg).

Reply:
if we take max it would be big enough for both and no need to use dynamic allocation
another option is to use ucs_alloca()
also, even if we know what are those constants on a particular development system, should not assume the same for other systems, and let compiler calculate ucs_max()